lib/veil/credential_collection/base.rb

```diff
@@ -85,6 +85,24 @@ def get(*args)
         end
       end
 
+      #
+      # Check to see if a given credential has been added.
+      #
+      def exist?(*args)
+        if maybe_get(*args).nil?
```

Comment:
[nit] This if statement could also be `!maybe_get(*args).nil?`... what's in the PR is more obvious, though.